src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AbtractAdapter.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+public class AbtractAdapter extends AbstractDriver {
+
+    private int startX, startY;
+
+    public AbtractAdapter() {
+        super(0, 0);
+    }
+
+    @Override public void operateTo(int x, int y) {
+        ILine line = LineFactory.getBasicLine();
+        line.setStartCoordinates(this.startX, this.startY);
```

Comment:
AbstractDriver already has some implementation, which should be used.
Direct SetPosition calls are ignored so this Adapter would not work well.